Emby.Server.Implementations/Services/StringMapTypeDeserializer.cs

```diff
@@ -110,9 +110,9 @@ public static string LeftPart(string strVal, char needle)
         }
     }
 
-    internal class TypeAccessor
+    internal static class TypeAccessor
```

Comment:
pls don't change stuff that is unrelated to main PR. at the very least edit the description... but I would really prefer separate PRs.